lib/test.js

```diff
@@ -217,7 +217,7 @@ Test.prototype._assert = function assert (ok, opts) {
     if (!ok) {
         var e = new Error('exception');
         var err = (e.stack || '').split('\n');
-        var dir = path.dirname(__dirname) + '/';
+        var dir = path.dirname(__dirname) + path.sep;
```

Comment:
Ah, I see that `path.join` will always omit the trailing separator. This is fine as-is.